README.md

```diff
@@ -1,71 +1,196 @@
 
-## Status
+<p align="center">
+  <img width="100" height="100" src="images/spokestack.png">
+</p>
 
+[![GitHub license](https://img.shields.io/github/license/spokestack/spokestack-python)](https://github.com/spokestack/spokestack-python/blob/master/LICENSE.txt)
 [![CircleCI](https://circleci.com/gh/pylon/streamp3.svg?style=shield)](https://circleci.com/gh/spokestack/spokestack-python)
 [![PyPI version](https://badge.fury.io/py/spokestack.svg)](https://badge.fury.io/py/spokestack)
 [![Coverage Status](https://coveralls.io/repos/github/spokestack/spokestack-python/badge.svg?branch=master)](https://coveralls.io/github/spokestack/spokestack-python?branch=master)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![Documentation Status](https://readthedocs.org/projects/spokestack/badge/?version=latest)](https://spokestack.readthedocs.io/en/latest/?badge=latest)
+[![GitHub stars](https://img.shields.io/github/stars/spokestack/spokestack-python?style=social)](https://github.com/spokestack/spokestack-python/stargazers)
+![GitHub watchers](https://img.shields.io/github/watchers/spokestack/spokestack-python?style=social)
+![Twitter Follow](https://img.shields.io/twitter/follow/spokestack?style=social)
+
+Welcome to Spokestack Python! This library is intended for developing [voice interfaces](https://en.wikipedia.org/wiki/Voice_user_interface) in Python. This can include anything from [Raspberry Pi](https://www.raspberrypi.org/) applications like traditional smart speakers to [Django](https://www.djangoproject.com/) web applications. *Anything* built in [Python](https://www.python.org/) can be given a voice interface.
 
 ## Get Started
 
 ### System Dependencies
 
+There are some system dependencies that need to be downloaded in order to install spokestack via pip.
+
 #### macOS
 
-    brew install lame portaudio
+```shell
+brew install lame portaudio
+```
 
 #### Debian/Ubuntu
 
-    sudo apt-get install portaudio19-dev libmp3lame-dev
+```shell
+sudo apt-get install portaudio19-dev libmp3lame-dev
+```
+
+#### Windows
+
+We currently do not support Windows 10 natively, and recommend you install [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with the Debian dependencies. However, if you would like to work on native Windows support we will glady accept pull requests.
+
+Another potential avenue for using `spokestack` on Windows 10 is from [anaconda](https://www.anaconda.com/). This is without support for Text To Speech (TTS) though due to the Lame dependency. PortAudio, on the other hand, can be installed via `conda`.
 
-Add voice capability to your projects with one line.
+```shell
+conda install portaudio
+```
 
-    pip install spokestack
+### Installation with pip
 
-## Development
+Once system dependencies have been satisfied, you can install the library with the following.
+
+```shell
+pip install spokestack
+```
+
+## Usage
 
 ### Setup
```

Comment:
The setup and TF sections feel more like part of `Get Started` rather than `Usage` since we're still talking about dependencies.